src/Entity.php

```diff
@@ -83,10 +90,9 @@ protected function request($method, $relativeUrl, $data = null)
     {
         $request = new Request();
 
-        $response = $request->request($method, $relativeUrl, $data);
+        $response = $request->request($method, $relativeUrl, $data, $this->additionHeader);
```

Comment:
why pass the additional header from create function if u are referencing this here